packages/wallets/src/features/cashier/flows/WalletFiatOnRamp/WalletFiatOnRamp.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useMemo } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useActiveWalletAccount } from '@deriv/api';
-import { FiatOnRampModule } from '../../modules';
+import { FiatOnRampModule } from '../../modules/FiatOnRamp';
```

Comment:
Why this change ? modules are already exporting FiatOnRamp

Reply:
Had an issue with importing FiatOnRampModule in the test. FIxed the implementation